docs/installation/network-options.md

```diff
@@ -209,11 +220,31 @@ systemctl restart k3s
 ```
 :::
 
-:::caution Warning
-The latency between nodes will increase as external connectivity requires more hops. This will reduce the network performance and could also impact the health of the cluster if latency is too high.
-:::
+### Tailscale
 
-:::caution Warning
-Embedded etcd will not use external IPs for communication. If using embedded etcd; all server nodes must be reachable to each other via their private IPs.
-:::
+Available in v1.27.3, v1.26.6, v1.25.11 and newer.
+
+K3s can integrate with [tailscale](https://tailscale.com/) so that nodes use the tailscale VPN network to communicate with each other.
+
+There are four steps to be done with tailscale before deploying k3s:
+1 - Create an account in tailscale
```

Comment:
```suggestion
There are four steps to be done with Tailscale before deploying k3s:
1 - Log in to your Tailscale account
```

Reply:
English question: `Log in to` or `log into`

Reply:
Maybe "Sign in"? I have seen it as "login to", " log in to", and "log into", but I think only the three word version is grammatically correct.